uber/templates/summary/shirt_manufacturing_counts.html

```diff
@@ -12,12 +12,8 @@
     We recommend adding in a few extra shirts for safety margins to these numbers.
 </p>
 
-<p>
-    <b>IMPORTANT NOTE: The STAFF SHIRT report shows the result of EACH staffer receiving {{ c.SHIRTS_PER_STAFFER }} shirt(s).</b>
-</p>
-
```

Comment:
any reason not to keep this note here?  first thing brent and Nick asked me was 'does this show the 2 or 1 shirts per staffer?' : )